README.MD

```diff
@@ -74,6 +74,12 @@ We use the following testing libraries:
 - [mocha](https://https://mochajs.org/), a well documented testing framework for javascript 
 - [chai](http://www.chaijs.com/), the preferred assertion library for testing
 
+You'll need at least:
+- `docker-compose` 1.12 or above
+- `nodejs` 8.11.x or above
+- `yarn` 1.7.0 or above
+
```

Comment:
so that would be "8.11 or above" and "1.7 or above"?

Reply:
Yes, small typo